concent_api/middleman/middleman_server.py

```diff
@@ -175,9 +182,12 @@ def _close_middleman(self) -> None:
         if self._is_signing_service_connection_active:
             writer.close()
         else:
-            self._is_signing_service_connection_active = True
             tasks: list = []
             try:
+                successful = await self._authenticate_signing_service(reader, writer)
+                if not successful:
+                    writer.close()
+                    return
```

Comment:
We don't have a test for this part, when authentication is not successful. Just saying :P